src/components/Banner.tsx

```diff
@@ -17,7 +18,13 @@ import Tooltip from './Tooltip';
 
 type BannerProps = {
     /** Text to display in the banner. */
-    text: string;
+    text?: string;
```

Comment:
Why did this become optional? A banner without text sounds weird.

Reply:
That's because I've added the `content` property for a case when I need to pass a react node, not a simple text.